crates/bevy_render/src/camera/camera.rs

```diff
@@ -326,6 +326,22 @@ impl RenderTarget {
         }
     }
 
+    /// Retrieves the [`TextureFormat`] of this render target, if it exists.
+    pub fn get_texture_format<'a>(
```

Comment:
I'm slightly too late, but shouldn't this be called something like `get_render_target_texture_format`? And we could probably drop the get prefixes from this and similar functions.

Reply:
no firm position on my side on naming, but don't hesitate to open follow up PRs 👍

Reply:
We sometimes use "get" when either: (1) an option is returned or (2) it isn't a direct property of the object. This sort of falls into both categories, but I don't have super strong opinions here.

On the topic of `get_render_target_texture_format`: the "render target" is implicit as this type is `RenderTarget`. We wouldn't rename `Window::position()` to `Window::window_position` for the same reason.